ranlab/README.md

```diff
@@ -0,0 +1,30 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
Will we need another README in this directory or should we move all instructions into root?

Reply:
Hmm, we should be able to move most of these files up one level 🤔 but we don't need this extra README, no

Reply:
The Next.js CLI demanded to create a project name without capitals BUT I can try moving these files around and seeing if everything still works fine. I agree that it should all be at root level. I have a feeling it will be ok.